n64.mk

```diff
@@ -102,7 +102,12 @@ RSPASFLAGS+=-MMD
 %.dfs:
 	@mkdir -p $(dir $@)
 	@echo "    [DFS] $@"
-	$(N64_MKDFS) $@ $(<D) >/dev/null
+	set -e; \
+	MKDFSROOT="$(shell python3 -c 'import os.path; print( os.path.commonpath( """$^""".split() ) ) ')"; \
```

Comment:
I don't love this, but every alternative I tried was worse. Can we safely assume Python exists?

Reply:
Obviously we cannot assume Python exists, as demonstrated by GitHub Actions. 😢